codebuild/cd/fips-compat-jar-build.yml

```
@@ -0,0 +1,34 @@
+version: 0.2
+#this buildspec assumes the aws-common-runtime/ubuntu-16.04 image
+# This job is responsible for artifacting the JAR which will have all of the other shared libs stuffed
+# into it once all platforms are built and artifacted
+phases:
+  install:
+    commands:
+      - sudo add-apt-repository ppa:openjdk-r/ppa
+      - sudo apt-get update -y
+      - sudo apt-get install openjdk-11-jdk-headless maven wget unzip -y -f
+  build:
+    commands:
+      - cd $CODEBUILD_SRC_DIR/aws-crt-java
+      # upload artifacts to S3
+      - export GIT_TAG=$(git describe --tags)
+      - mkdir -p target/cmake-build/lib
+      # prepare fips-compat uber jar, download the regular libs
+      - aws s3 cp --recursive s3://aws-crt-java-pipeline/${GIT_TAG}/lib $CODEBUILD_SRC_DIR/aws-crt-java/target/cmake-build/lib
+      # Override with the FIPS libs
+      - aws s3 cp --recursive s3://aws-crt-java-pipeline/${GIT_TAG}/fips_lib $CODEBUILD_SRC_DIR/aws-crt-java/target/cmake-build/lib
+      # Run a test to make sure we get the FIPS libs
+      - export CRT_FIPS=ON
+      - mvn test -Dtest=software.amazon.awssdk.crt.test.SystemInfoTest -Dshared-lib.skip=true
```

trivial: cool! but let's be a bit more explicit
```suggestion
      - mvn test -Dtest=software.amazon.awssdk.crt.test.SystemInfoTest.testIsFIPS -Dshared-lib.skip=true
```